core/cautils/getter/gcpcloudapi.go

```diff
@@ -0,0 +1,39 @@
+package getter
+
+import (
+	"context"
+
+	containeranalysis "cloud.google.com/go/containeranalysis/apiv1"
+)
+
+type GCPCloudAPI struct {
+	credentialsPath string
+	context         context.Context
+	client          *containeranalysis.Client
+	projectID       string
+	credentials     bool
+	loggedIn        bool
```

Comment:
I don't see anywhere in the code that we set the values of these 3 fields, referring to `projectID` , `credentials` and `loggedIn`...

Reply:
thats where credentials come into picture and I am not aware how kubescape adds credentials for the armosec adaptor so I have kept that open for now

Reply:
I think as a first step we can use environment variables for that. In that case I believe that `credentials` will have to be of type `string` rather than `bool`
Could you please a README file which explains on this adaptor / how to configure it? with a short example.

Reply:
adding readme would be a great idea, will add it in some time